queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-02-12' AND '2019-05-07';
```

Comment:
- [ ] Kindly notice that you should list the name of all animals born between 2016 and 2019, so you should write the dates correctly.

```sql
SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Reply:
Thanks @ShahierNashaat , can you please give me a hint on how to do that, I tried `SELECT name from animals WHERE date_of_birth BETWEEN '2016%' AND '2019%';` but it does not work but it works when I tried it from another editor

Reply:
Oh! I just saw that you indeed gave me hint. Thanks!